src/App.js

```diff
@@ -1,17 +1,42 @@
 import React from 'react';
+import TodoItem from './components/TodoItem';
+import InputField from './components/InputField'
 import './App.css';
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  };
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: [
+                {text: 'Random text', id: 0},
+                {text: 'One more random text', id: 1}
+            ]
+        };
+    };
+    handleEnterPress = (text) => {
+        const id = this.state.data.length;
```

Comment:
Try to think why this way of id generation is not good.

Reply:
That's temporary way, i'll use mongo _id when connect with backend